.eslintignore

```diff
@@ -0,0 +1,3 @@
+dist/
```

Comment:
I've normally seen the targeting of what to lint in the lint script itself vs this ignore, but that doesn't mean this is wrong.  Is this also so you can run eslint from the cli without knowing the incantation of args to run it?

Fine with this, just curious.

Reply:
Yeah, the main reason I did it actually is that it's useful for running the vscode eslint plugin so it will ignore these